packages/compiler-core/src/transforms/transformElement.ts

```diff
@@ -331,12 +331,12 @@ function resolveSetupReference(name: string, context: TransformContext) {
     if (bindings[name] === type) {
       return name
     }
-    if (bindings[camelName] === type) {
-      return camelName
-    }
     if (bindings[PascalName] === type) {
       return PascalName
     }
+    if (bindings[camelName] === type) {
```

Comment:
This is not the right fix.
consider:
```js
import viewComponent from '/src/components/ViewComponent.vue' // v is lowercase
async function ViewComponent(index) { // V is uppercase

}
```

Reply:
Thanks for your review, I will try to modify it.

Reply:
But after thinking about it, I still think that such a tag should match `PascalName`, and the editor seems to match it in this way.

![image](https://github.com/user-attachments/assets/3f9aaab0-cd0a-4296-91fb-dd5319b6cafc)

Reply:
Yes, PascalName is recommended in [the document](https://vuejs.org/guide/components/registration.html#component-name-casing), but this actual a breaking change.

- [playground](https://play.vuejs.org/#eNp9UstOwzAQ/JWVLwWpJAc4VRESoB7gAAgQJ1+qdFNckrXlR1opyr+zdppQCdqbd2bsnZ11J+6MydqAYiEKV1plPDj0wdxKUo3R1kMHFivoobK6gRlLZxPVKtw96MYcuCyPRXyNJZKqQKVXmuDzILu47CQBP+eDJZiNMIt7SUU+tOfGXHhsTL3yyBVAEftclSzNuS7yiRRz4V2pqVKbbOs08RCpgxRRrGq0LyY6cFIsIDGRW9W13j0lzNuA8xEvv7D8/gffun3EpHi16NC2KMXE+ZXdoB/o5fsz7vk8kY1eh5rVZ8g3dLoO0eMguw+0ZttHuuT2MeWtaPPhlnuP5MahotGo7JNeCs4+Rnpq9F+719lNusfZc4rj3v78gnNbWas2HQBi2gnKB+xoRZJE/wO8+c42)
- [playground with this PR](https://deploy-preview-11438--vue-sfc-playground.netlify.app/#eNp9UstOwzAQ/JWVLwWpJAc4VRESoB7gAAgQJ1+qdFNckrXlR1opyr+zdppQCdqbd2bsnZ11J+6MydqAYiEKV1plPDj0wdxKUo3R1kMHFivoobK6gRlLZxPVKtw96MYcuCyPRXyNJZKqQKVXmuDzILu47CQBP+eDJZiNMIt7SUU+tOfGXHhsTL3yyBVAEftclSzNuS7yiRRz4V2pqVKbbOs08RCpgxRRrGq0LyY6cFIsIDGRW9W13j0lzNuA8xEvv7D8/gffun3EpHi16NC2KMXE+ZXdoB/o5fsz7vk8kY1eh5rVZ8g3dLoO0eMguw+0ZttHuuT2MeWtaPPhlnuP5MahotGo7JNeCs4+Rnpq9F+719lNusfZc4rj3v78gnNbWas2HQBi2gnKB+xoRZJE/wO8+c42)